homeassistant/components/adguard/sensor.py

```diff
@@ -178,7 +178,7 @@ def __init__(self, adguard):
         """Initialize AdGuard Home sensor."""
         super().__init__(
             adguard,
-            "Searches Safe Search Enforced",
```

Comment:
If a user is using unique IDs, this won't be a breaking change as the previous entity ID will remain the same for existing installations.

Reply:
Okey, but still need to be noticed ?

Reply:
This is not a breaking change.

Reply:
Removing breaking change